nsxt/resource_nsxt_vpc_service_profile.go

```diff
@@ -24,14 +24,6 @@ var vpcServiceProfileModeValues = []string{
 	model.VpcProfileDhcpConfig_MODE_DEACTIVATED,
 }
 
-var vpcServiceProfileLogLevelValues = []string{
```

Comment:
Was this really unused?

Reply:
Yes, these values were also removed from the SDK